cmd/app/bootstrap/runner.go

```diff
@@ -420,43 +486,35 @@ func (r *runner) commitAndPush(ctx context.Context, repoPath string) error {
 		return microerror.Mask(err)
 	}
 
-	err = r.execCommand(ctx, repoPath, "git", "push", "origin", branchName)
+	err = r.execCommand(ctx, repoPath, "git", "push", "origin", "main")
```

Comment:
Makes sense for the first commit. And this doesn't cause a notification for smbd listed in CODEOWNERS?

Reply:
Could there ever be a template with another default branch name than `main`?

Reply:
I don't think so. We can adapt the code if necessary then.